### ISSUE TITLE
[7.0.1] Missing types on import

### ISSUE THREAD

**Issue by spinlud** [puppeteer/puppeteer#6829]
* Puppeteer version: 7.0.1
* Node.js version: 12.x

```ts
import { Browser, BrowserContext, Request } from 'puppeteer';  // <--- Missing types
```

With the new version, what is the right way to import puppeteer types?

**Comment by SProst** [comment]
This is possibly a duplicate of #6816 and tangentially related to #6820.

**Comment by fardolieri** [comment]
This seems to be fixed with version 7.0.4

**Comment by spinlud** [comment]
[7.0.4] Seems now it can't find only Request:

![image](https://user-images.githubusercontent.com/12783208/107381156-de57f880-6aee-11eb-9b7c-289755a8eff6.png)
